styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -120,12 +121,14 @@
   public static final String STYX_MODE_DEVELOPMENT = "development";
 
   public static final int SCHEDULER_TICK_INTERVAL_SECONDS = 2;
+  public static final int SUBMISSION_RATE_LIMITER_UPDATE_TICK_INTERVAL_SECONDS = 5;
```

Comment:
I think it could definitely make sense to refactor how runtime configuration is handled. Not sure atm how that should look though. Perhaps a separate PR?